src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/MyApp1/.template.config/template.json

```
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [ "Common", "AI", "Web" ],
+  "identity": "Microsoft.Extensions.AI.Templates.Chat.CSharp",
+  "name": "Chat with Custom Data",
```

Should AI be in the name? eg
```suggestion
  "name": "AI Chat with Custom Data",
```

---

Done